### PR TITLE
try ninja again

### PR DESCRIPTION
**Pull request by RikNieuwoudt**
@casey-collab

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2313)

### CODE REVIEW

File: english/buy-gameboy/buy-gameboy.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 you realize that Pokemon is great and can still help you with your troubles, 
 you find a man at a flea market selling Pokemon Red,
-Blue, and Yellow all for 20 dollars along with a Gameboy. You buy it. 
\ No newline at end of file
+Blue, and Yellow all for 20 dollars along with a Gameboy. You buy it. 
+
+[english/ninja/ninja.md](but this isn't a game, boy.)
```
Comment (casey-collab) [inline english/buy-gameboy/buy-gameboy.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/ninja/ninja.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+[english/buy-gameboy/buy-gameboy.md](gameboy)
```
Comment (casey-collab) [inline english/ninja/ninja.md]: This link is also invalid.

Comment (casey-collab) [inline english/ninja/ninja.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by RikNieuwoudt** [comment]
@casey-collab

File: english/buy-gameboy/buy-gameboy.md [hunk 3]
```diff
@@ -1,3 +1,5 @@
 you realize that Pokemon is great and can still help you with your troubles, 
 you find a man at a flea market selling Pokemon Red,
-Blue, and Yellow all for 20 dollars along with a Gameboy. You buy it. 
\ No newline at end of file
+Blue, and Yellow all for 20 dollars along with a Gameboy. You buy it. 
+
+[But it's not a game, boy](./english/ninja/ninja.md)
```
Comment (casey-collab) [inline english/buy-gameboy/buy-gameboy.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/ninja/ninja.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+[from buy-gameboy](./english/buy-gameboy/buy-gameboy.md)
```
Comment (casey-collab) [inline english/ninja/ninja.md]: This link is also invalid.

Comment (casey-collab) [inline english/ninja/ninja.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by RikNieuwoudt** [comment]
@casey-collab

File: english/buy-gameboy/buy-gameboy.md [hunk 5]
```diff
@@ -1,3 +1,5 @@
 you realize that Pokemon is great and can still help you with your troubles, 
 you find a man at a flea market selling Pokemon Red,
-Blue, and Yellow all for 20 dollars along with a Gameboy. You buy it. 
\ No newline at end of file
+Blue, and Yellow all for 20 dollars along with a Gameboy. You buy it. 
+
+[But it's not a game, boy](/english/ninja/ninja.md)
```
Comment (casey-collab) [inline english/buy-gameboy/buy-gameboy.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/ninja/ninja.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
+my game
```
Comment (casey-collab) [inline english/ninja/ninja.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/buy-gameboy/buy-gameboy.md (hunk 5 above)
Comment (cbuckey-uda) [inline english/buy-gameboy/buy-gameboy.md]: Sorry about this, Casey doesn't understand this type of link. I just checked, and it's actually working correctly.

**Comment by cbuckey-uda** [comment]
Looks good to me! :video_game: :boy:

**Comment by RikNieuwoudt** [comment]
Thanks, Caroline! And thanks for the tutorial, saved me a lot of time.

**Pull request merged**